tests/components/octoprint/test_button.py

```diff
@@ -192,3 +192,78 @@ async def test_stop_job(hass: HomeAssistant) -> None:
         )
 
         assert len(stop_command.mock_calls) == 0
+
+
+async def test_shutdown_system(hass: HomeAssistant) -> None:
+    """Test the shutdown system button."""
+    await init_integration(hass, BUTTON_DOMAIN)
+
+    coordinator: OctoprintDataUpdateCoordinator = hass.data[DOMAIN]["uuid"][
+        "coordinator"
+    ]
+
+    # Test shutting down the system
+    with patch("pyoctoprintapi.OctoprintClient.shutdown") as shutdown_command:
```

Comment:
You need to patch where this is getting used. i.e. homeassistant.components.octoprint.coordinator.OctoprintClient.shutdown

Or something similar

You should avoid doing global patches